hoomd/ParticleData.h

```diff
@@ -1242,11 +1242,13 @@ class PYBIND11_EXPORT ParticleData
         void setGPUAdvice();
     };
 
-class PYBIND11_EXPORT LocalParticleData : public LocalDataAccess<ParticleData>
+template<class OUTPUT>
```

Comment:
Use `Output`. All-caps name are for constants.

Reply:
Will do, I was under the assumption that template parameters should be all caps (probably because in learning I never saw one with more than one letter.